src/_listDefinitions.scss

```diff
@@ -93,6 +94,7 @@ $semantic-color: (
       left: 0.125rem;
       right: 0.125rem;
       bottom: 0.125rem;
+      pointer-events: none;
```

Comment:
Can you explain what you are achieving with enabling/disabling pointer events here?

Reply:
this fix was done by markowski in core Ngx at https://github.com/SAP/fundamental-ngx/blob/main/libs/core/src/lib/list/list.component.scss#L4. i am not very sure of the reason. i have moved it here from Ngx.

Reply:
when navigation list item is in focus, cursor is not changing to "grabbing cursor". by making pointer-events: none, it has "grabbing" cursor.
![list-cursor-pointer](https://user-images.githubusercontent.com/57661487/119135568-3a220e00-ba5c-11eb-8fb9-9d6ceeb8ec4d.gif)